fw_gear_hcp_diff/args/DiffPreprocPipeline.py

```diff
@@ -0,0 +1,383 @@
+"""
+Builds, validates, and excecutes parameters for the HCP script
+/opt/HCP-Pipelines/DiffusionPreprocessing/DiffPreprocPipeline.sh
```

Comment:
Just to confirm this is not written by us, this is supplied by the HCP?

Reply:
I believe so.